### DISCUSSION TITLE
Proposed change to keyboard shortcuts

### DISCUSSION THREAD

**Discussion by Vigasaurus** [plausible/analytics#734] [Anything Else]
I've been thinking about it, and I feel like in hindsight we shouldn't have used `3` and `6` for `30 days` and `6 months` keyboard shortcuts, respectively, and instead assign `1-9` as ways to quickly swap between sites. In other words, the same concept as the date picker, but for the site switcher. I know that this could potentially cause issues for those with >9 sites, but I figure most people have less than that, and even those with more could benefit from some shortcuts. (And maybe in the future, some re-ordering could be done to facilitate 9 top sites, in a way). 

I figure different keybinds for 30 day and 6 month could be thought up (maybe `T` and `S` or similar), and the numbers reserved for this. 

Something like the following (ofc not real styling, just a mockup):
![image](https://user-images.githubusercontent.com/19434157/108576396-1c210200-72e3-11eb-9a7f-52a62f3e8304.png)

Thoughts?

**Comment by metmarkosaric** [comment]
I like that @Vigasaurus! 3 and 6 was more to make it easier to remember 30 days and 6 months but this makes more sense to do. I have no specific keys in mind to replace 3 and 6 so whatever you feel is natural is ok for me. I'm already fully using the existing shortcuts and it makes it so much faster to explore the dashboard. Numbers for sites would improve the speed even more! Thanks!

**Reply by Vigasaurus** [reply]
Sweet! I figured this would be a welcome addition - more intuitive keybindings are always great imo. For now I think I'll do `t`hirty days, and `s`ix months - but that is obviously easy to change if anyone else/uku have differing ideas for good keybinds.

**Reply by metmarkosaric** [reply]
Sounds good to me!
